### ISSUE TITLE
NTR: Giardia antigen present on immune assay or DFA of stool (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7921]
New term request Giardia antigen present on immune assay or DFA of stool (nadea3_190513014245): EIA; direct fluorescent antibody; fecal

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000324
- label: Positive stool Giardia antigen test
- parents: [Positive stool infectious agent test (HP:0034866)](https://hpo.jax.org/app/browse/term/HP:0034866)
- pmid: PMID:12574257
- def: Fecal analysis for Giardia antigen is abnormal. Tests include direct fluorescent-antibody (DFA), which detect sintact organisms, and enzyme immunoassays (EIAs), which detect soluble stool antigens.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7921
- synonyms:

**Comment by MickeySegal** [comment]
It seems there was another HPO term assigned previously: https://hpo.jax.org/app/browse/term/HP:0034867
